src/creature.h

```diff
@@ -544,6 +544,9 @@ class Creature
         virtual const std::string &symbol() const = 0;
         virtual bool is_symbol_highlighted() const;
 
+		// Turn since creature was milked for last time
+		int turn_next_milking = 0;
```

Comment:
I just realize that I need to add some variable for milk charges left, in case you don't milk all the milk out of the udders.
I'm thinking of milking animals like toilets with milk instead of water.